tests/main_test.py

```diff
@@ -190,6 +188,11 @@ def test_get_tts_server_lists(_, fetch_audio_data, tts_instance_factory):
     custom_hosts = ["https://customhost1.com", "https://customhost2.com"]
     tts_instance = tts_instance_factory(config={"host": custom_hosts})
     tts_instance.get_tts("test", "test.wav")
+    # Ensure custom list is NOT shuffled
+    i = 0
+    for host in custom_hosts:
+        assert custom_hosts[i] == tts_instance.host[i]
+        i += 1
```

Comment:
This works, but for reference, Python has an `enumerate()` function that has less chance of off-by-one errors:

```py
for i, host in enumerate(custom_hosts):
    assert custom_hosts[i] == tts_instance.host[i]
```